src/scss/qld-type.scss

```diff
@@ -109,26 +115,22 @@ strong, b, th {
 
 
 // Light, Light alt
-a {
-  font-weight: var(--qld-link-font-weight);
-}
 
 a,
 a.nav-link {
-  text-decoration: underline;
-  text-underline-offset: var(--qld-link-underline-offset) !important;
-  text-decoration-thickness: var(--qld-link-underline-thickness) !important;
+  text-underline-offset: var(--qld-link-underline-offset);
+  text-decoration-thickness: var(--qld-link-underline-thickness) !important;  // Overrides default BS styles
 
   &:hover {
-    text-decoration-thickness: var(--qld-link-underline-thickness-hover) !important;
+    text-decoration-thickness: var(--qld-link-underline-thickness-hover) !important; // Overrides default BS styles
```

Comment:
I'm curious if we can override BS5 without an !important tag.
Totally understand if timing is tight atm.

Reply:
I wasn't able to find where, and wasn't sure why this didn't work as an override without the !important. Keen if anyone can shed some light.

Reply:
In theory we should be able to rescope a bootstrap CSS variable...like --text-decoration-thickness or similar.  But  Bootstrap might not have a variable setup for that yet... I know it's ongoing part of their project to swap out hard coded values with variables but still a few gaps.

Failing that, just the fact that our CSS is rendered _after_ the bootstrap CSS should mean our CSS is used by the browser.   Bootstrap deliberatly removed all of their !important tags for this reason, so when a rule is redefined further along in the CSS, that later rule is honoured.

Reply:
You might have been competing with this one @re-becca?  Was it inside sidenav?

![image](https://github.com/qld-gov-au/qgds-bootstrap5/assets/126438309/487052c7-a6d9-489c-bf30-b9b499f92f40)

Reply:
@stvp-qld It's coming from Bootstrap's reboot.scss. 'text-decoration: underline' being defined there somehow then prevents 'text-decoration-thickness' being applied without the !important
<img width="533" alt="Screenshot 2024-06-20 at 1 08 03 pm" src="https://github.com/qld-gov-au/qgds-bootstrap5/assets/950889/ada442dc-094c-4004-b7d6-6bc8d491e4be">
This is across any component that inherits 'a' styles - cards, typography.

Reply:
Ah looks like 'reboot.scss' is imported after 'qld-type.scss' unlike components.scss, will chat to you @stvp-qld  about how best to handle this